resources/lang/deu.ini

```diff
@@ -0,0 +1,100 @@
+; NOTE: We use the word newspaper for the following two meanings: the newspaper office and the newspaper (reading).
+
+; This string is used with ~.perm.* strings
+main.perm.base=Du hast nicht die Berechtigung für {%0}
```

Comment:
I don't speak German so I'm not sure, but is this following the translation of commands.generic.permission from Minecraft? For your better understanding, like mentioned from the above, this string is used with other permission strings like main.perm.generic, gui.buyinfo.perm.buy etc.

Original English string (from MCJE): You do not have permission to use this command
German translation (from MCJE): Du hast nicht die benötigte Berechtigung, diesen Befehl auszuführen

If it is following the style of translation from Minecraft then it's okay, but if it isn't, could you change your translation to follow it? Thanks!

Reply:
Both mine and the official MCJE translation are perfectly fine, however I´d consider the official one slightly more formal. I agree to use that instead!

Reply:
I just left out the german word for "command" and "to use" as in "You don't have permission to {%0}" instead of "You do not have permission to use this command"